doc/cug.tex

```diff
@@ -4551,6 +4489,380 @@ \subsubsection{Collapsible Families In Suite Graphs}
 \label{fig-namespaces}
 \end{figure}
 
+\subsection{Parameterized Tasks}
+\label{Parameterized Tasks}
+
+Cylc can automatically generate tasks and dependencies by expanding
+parameterized task names over lists of parameter values.  Uses include:
+\begin{myitemize}
+    \item generating an ensemble of similar model runs
+    \item generating chains of tasks to process similar datasets
+    \item replicating an entire workflow, or part thereof, over several runs
+    \item splitting a model run into steps or ``chunks`` without dynamic cycling
+\end{myitemize}
+
+{\em Note that this can be done with Jinja2 loops too (Section~\ref{Jinja2})
+    but parameterization is much cleaner: nested loops can seriously reduce
+the clarity of a suite definition.}
+
+\subsubsection{Parameter Expansion}
+
+Parameters values can be lists of strings, or integer ranges (both bounds included):
+\begin{lstlisting}
+[cylc]
+    [[parameters]]
+        obs = ship, buoy, plane
+        run = 1..5  # 1, 2, 3, 4, 5
+\end{lstlisting}
+
+Angle brackets denote parameterization.  For example,
+\begin{lstlisting}
+    model<run>  # for run = 1..2
+\end{lstlisting}
+expands to,
+\begin{lstlisting}
+    model_run1, model_run2
+\end{lstlisting}
+and,
+\begin{lstlisting}
+    proc<obs>  # for obs = ship, buoy, plane
+\end{lstlisting}
+expands to,
+\begin{lstlisting}
+    proc_ship, proc_buoy, proc_plane
+\end{lstlisting}
+For integer values the parameter name appears in the expanded task name, but
```

Comment:
It's not a limitation, it's a feature.   If you have parameters `m` and `n` (say) with integer values, the values alone don't tell you which is which, so by default we expand `foo<m,n>` to `foo_m1_n2` (etc.) rather than  `foo_1_2` (does the `2` come from `m` or `n`?). 

I will make this clearer though.